src/export-page/utils.js

```diff
@@ -0,0 +1,16 @@
+import Cookies from 'universal-cookie';
+import moment from 'moment';
+
+import { LAST_EXPORT_COOKIE_NAME } from './data/constants';
+
+// eslint-disable-next-line import/prefer-default-export
+export const setExportCookie = (date, completed) => {
```

Comment:
For all functions in the utilities, we decided to add special [JSDocs](https://jsdoc.app/about-getting-started.html) documentation. For [example](https://github.com/raccoongang/frontend-app-course-authoring/blob/2u-master/src/advanced-settings/utils.js)